src/hb-ot-glyf-table.hh

```diff
@@ -1072,18 +1072,8 @@ struct glyf
       if (!get_offsets (glyph, &start_offset, &end_offset))
 	return false;
 
-      if (end_offset - start_offset < GlyphHeader::static_size)
-	return true; /* Empty glyph; zero extents. */
-
-      const GlyphHeader &glyph_header = StructAtOffset<GlyphHeader> (glyf_table, start_offset);
-
-      /* Undocumented rasterizer behavior: shift glyph to the left by (lsb - xMin), i.e., xMin = lsb */
-      /* extents->x_bearing = hb_min (glyph_header.xMin, glyph_header.xMax); */
-      extents->x_bearing = hmtx_accel.get_side_bearing (glyph);
-      extents->y_bearing = hb_max (glyph_header.yMin, glyph_header.yMax);
-      extents->width     = hb_max (glyph_header.xMin, glyph_header.xMax) - hb_min (glyph_header.xMin, glyph_header.xMax);
-      extents->height    = hb_min (glyph_header.yMin, glyph_header.yMax) - extents->y_bearing;
-
+      hb_bytes_t ((const char *) glyf_table + start_offset,
+		  end_offset - start_offset).as<GlyphHeader> ()->get_extents (extents);
```

Comment:
https://github.com/harfbuzz/harfbuzz/blob/ae2c4c9d2db196bf1d0b6e8b58f6cbe52f0240c8/src/hb-ot-glyf-table.hh#L296 isn't doing the same, will submit I PR to the branch